CODING_GUIDELINES.md

```diff
@@ -87,10 +89,14 @@ default:
 }
 ```
 
-#### d. single-line blocks (lambdas, initializer lists etc.) ####
+#### d. Brace enclosed initializers ####
+Unlike single-line functions, you must not insert spaces between the brackets and concluded expressions.<br />
+But you must insert a space between the variable name and initializer.
 ```c++
- {} // empty - space before {
- { body } // spaces around { and before }
+Class obj {}; // empty
+Class obj {expr};
+Class obj {expr1, /*...,*/ exprN};
+QVariantMap map {{"key1", 5}, {"key2", 10}};
 ```
```

Comment:
Did I miss something? I don't think spaces were talked in @evsh's issue.
Isn't 
`QVariantMap map { { "key1", 5 }, { "key2", 10 } };`
better than
`QVariantMap map {{"key1", 5}, {"key2", 10}};` ?
(and the examples above)